### PR TITLE
Add cadd module

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

This PR attempts to CADD module. 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by ramprasadn** [comment]
@nf-core/maintainers This is a bit of a weird one because CADD requires a massive (approx. 150GB) database to work, and I don't know how we can test this module without it. So the "test" for this module only is essentially a stub-run at this point. However, I tested it locally and with proper files and it works. What do you guys think? Can we make an exception for this module in this repo, or should I keep it as a local module in my pipeline? 😅

File: modules/nf-core/cadd/main.nf [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    label 'process_medium'
+
+    conda "bioconda::cadd-scripts=1.6 anaconda::conda=4.14.0 conda-forge::mamba=1.4.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-8d145e7b16a8ca4bf920e6ca464763df6f0a56a2:d4e457a2edecb2b10e915c01d8f46e29e236b648-0':
+        'quay.io/biocontainers/mulled-v2-8d145e7b16a8ca4bf920e6ca464763df6f0a56a2:d4e457a2edecb2b10e915c01d8f46e29e236b648-0' }"
+
+    containerOptions {
+        (workflow.containerEngine == 'singularity') ?
+            "--writable -B ${annotation_dir}:/usr/local/share/cadd-scripts-1.6-1/data/annotations" :
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply modules/nf-core/cadd/main.nf:12]: Yupe.. it is.

**Comment by jfy133** [comment]
Stub is fine if physcially can't test it on CI actions, and it definitely works locally :)

File: tests/modules/nf-core/cadd/test.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## TODO nf-core: Please run the following command to build this file:
```
Comment (sateeshperi) [inline tests/modules/nf-core/cadd/test.yml:1]:
```suggestion
```

File: tests/modules/nf-core/cadd/test.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+#                nf-core modules create-test-yml cadd
```
Comment (sateeshperi) [inline tests/modules/nf-core/cadd/test.yml:2]:
```suggestion
```

**Review by sateeshperi — COMMENTED** [review]
lgtm

**Comment by ramprasadn** [comment]
Thanks for the review @sateeshperi.. Good catch 😅 
I have updated the meta.yml file now 😄

**Review by sateeshperi — APPROVED** [review]
lgtm

**Pull request merged**